src/EnergyPlus/SetPointManager.cc

```diff
@@ -4707,14 +4716,16 @@ namespace SetPointManager {
 			SetTempAtOutHigh = this->OutHighSetPt1;
 		}
 
-		SetPt = CalcSetPoint(OutLowTemp, OutHighTemp, OutDryBulbTemp, SetTempAtOutLow, SetTempAtOutHigh);
-
-		if ( present( InitFlag ) ) {
-			Node( NodeNum ).TempSetPoint = SetPt; //Setpoint for Initial Routine
+		if ( CtrlType == iCtrlVarType_MaxTemp ) {
+			SetPt = SetTempAtOutLow;
+		} else if ( CtrlType == iCtrlVarType_MinTemp ) {
+			SetPt = SetTempAtOutHigh;
 		} else {
```

Comment:
These lines aren't needed anymore.  The calculated setpoint is the same, no matter what the control variable is.